boards/FSM/Test/Src/Test_AcceleratorPedal.cpp

```diff
@@ -6,15 +6,20 @@ extern "C"
 }
 
 FAKE_VALUE_FUNC(bool, is_encoder_alarm_active);
+FAKE_VOID_FUNC(reset_apps_encoder_counter);
+FAKE_VALUE_FUNC(uint32_t, get_apps_encoder_counter);
 
 class AcceleratorPedalTest : public testing::Test
 {
   protected:
     void SetUp() override
     {
-        accelerator_pedal =
-            App_AcceleratorPedal_Create(is_encoder_alarm_active);
+        accelerator_pedal = App_AcceleratorPedal_Create(
+            is_encoder_alarm_active, reset_apps_encoder_counter,
+            get_apps_encoder_counter, 0);
```

Comment:
Set the encoder max pressed value here to 0 as its not being used for any of the tests below. The accelerator pedal created also isn't referring specifically to the primary or secondary APPS

Reply:
@garethellis0 We are checking here if the alarm for a general accelerator pedal was working. Just wondering what you think we should be doing with this test now that we have a AcceleratorPedals containing the specific papps and sapps.